lighthouse-cli/bin.js

```diff
@@ -5,51 +5,57 @@
  */
 'use strict';
 
-import {existsSync} from 'fs';
-import * as path from 'path';
+const existsSync = require('fs').existsSync;
+const path = require('path');
 
-import * as Commands from './commands/commands';
-import * as Printer from './printer';
-import {getFlags} from './cli-flags';
-import {runLighthouse} from './run';
+const commands = require('./commands/commands.js');
+const printer = require('./printer.js');
+const getFlags = require('./cli-flags.js').getFlags;
+const runLighthouse = require('./run').runLighthouse;
 
 const log = require('lighthouse-logger');
+// @ts-ignore
```

Comment:
for some reason typescript won't resolve `json` files as part of `require()` (you get a `"Cannot find module '../lighthouse-core/config/perf.json'"` here). Searching didn't find a better solution, and @JoelEinbinder [did this in puppeteer](https://github.com/GoogleChrome/puppeteer/blob/f5bb333cd0c42306f47e7767e5cb8b78af557c27/lib/Launcher.js#L27), so I'm going with that unless anyone has a better fix :)

Reply:
did some searching as well, there is no support.